curator/run_cron.py

```diff
@@ -42,9 +42,9 @@
 #allowed_units = {'hours': 'hours', 'days': 'days', 'weeks': 'weeks', 'months': 'months'}
 allowed_units = {'days': 'days', 'weeks': 'weeks', 'months': 'months'}
 
-# allowed operations, currently we'll just allow delete
-allowed_operations = {'delete': 'delete'}
-curator_settings = {'delete': {}}
+# allowed operations, currently we'll just allow delete, close, and optimize
+allowed_operations = {'delete': 'delete', 'close': 'close', 'optimize': 'optimize'}
```

Comment:
Because that's the simplest way to create a lookup table.  Do you have a better suggestion?